pkg/auth/requests/impersonate_test.go

```diff
@@ -3,7 +3,9 @@ package requests
 import (
 	"context"
 	"errors"
+	"io"
 	"net/http"
+	"net/http/httptest"
 	"testing"
 
 	"github.com/golang/mock/gomock"
```

Comment:
nit:  maybe update this to the new, supported version of gomock go.uber.org/mock/gomock

Reply:
> nit: maybe update this to the new, supported version of gomock go.uber.org/mock/gomock

Not sure why it linked my suggestion to the httptest import....obviously, this is about github.com/golang/mock/gomock

Reply:
done